reference/media-type-collections.md

```diff
@@ -3,7 +3,8 @@ layout: page
 title: MediaCollection
 ---
 
-*NOTE*: This class is deprecated and will be removed in a future version of Praxis! You should <insert stuff about what to do instead>.
```

Comment:
let's fix the <insert stuff....> text, by adding something like:

You should convert any `MediaTypeCollections` to conform to `Praxis::Collection` (and summary types) as described in [Media Types](/reference/media-types/)